<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="51589" started-at="2016-04-19T22:29:37Z" finished-at="2016-04-19T22:30:28Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="51589" started-at="2016-04-19T22:29:37Z" finished-at="2016-04-19T22:30:28Z">
      <class name="testImage.TestNusringNITUX">
        <test-method status="PASS" signature="beforeMethod()[pri:0, instance:testImage.TestNusringNITUX@47f6473]" name="beforeMethod" is-config="true" duration-ms="10931" started-at="2016-04-19T18:29:37Z" finished-at="2016-04-19T18:29:48Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method status="FAIL" signature="test1()[pri:0, instance:testImage.TestNusringNITUX@47f6473]" name="test1" duration-ms="39556" started-at="2016-04-19T18:29:48Z" finished-at="2016-04-19T18:30:27Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at testImage.TestNusringNITUX.clickOnScheduleanIntegratedTest(TestNusringNITUX.java:64)
	at testImage.TestNusringNITUX.test1(TestNusringNITUX.java:430)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test1 -->
        <test-method status="PASS" signature="afterMethod()[pri:0, instance:testImage.TestNusringNITUX@47f6473]" name="afterMethod" is-config="true" duration-ms="1044" started-at="2016-04-19T18:30:27Z" finished-at="2016-04-19T18:30:28Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
      </class> <!-- testImage.TestNusringNITUX -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
